src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
@AlanFoster @rolyatmax check the last commit.
I've added the changes of #56 + isolated the `moment` object used by `CalendarMonth`. So now `require('moment/locale/${language}')` anywhere in the app, won't change the locale of the datepicker. Locale is passed as a prop and it will load the locale, set the locale of the moment object used by the datepicker and trigger a re-render.